azure-pipelines.yml

```diff
@@ -126,7 +126,33 @@ stages:
       condition: always()
       inputs:
         pathToPublish: $(Build.SourcesDirectory)/artifacts/log/$(BuildConfiguration)/
-        artifactName: BuildLogs
+        artifactName: BuildLogs-Linux-$(BuildConfiguration)
+        artifactType: container
+
+  - job: MacOS
+    displayName: MacOS Build
+    pool:
+      vmImage: macOS-latest
+
+    strategy:
+      matrix:
+        Build_Release:
+          BuildConfiguration: Release
+        Build_Debug:
+          BuildConfiguration: Debug
+
+    steps:
+    - script: ./build.sh --ci
```

Comment:
Not sure why this `build.sh` is in the project root while other scripts are in `/eng/common`.
It does not depend on this PR, just asking and wondering whether it can be moved away from the root to avoid noise on the main page.

Reply:
Everything under eng/common are all the scripts coming from arcade (our big build infrastructure) and everytime we ingest a new version of arcade, the contents of that folder get overridden. The build scripts vary a bit in a per-repo basis which is why all dotnet repos have them on the root (as well as for convenience)